lib/notification/apsProperties.js

```diff
@@ -122,6 +122,12 @@ module.exports = {
     }
   },
 
+  set interruptionLevel(value) {
+    if(typeof value === "string" || value === undefined) {
```

Comment:
```suggestion
    if (typeof value === "string" || value === undefined) {
```

nit: use the same spacing used elsewhere